_projects/2023/100481838/100481838.Rmd

```diff
@@ -0,0 +1,1067 @@
+---
+title: "We feel it already. How about we visualize it?"
+description: |
+  A short description of the post.
+categories: "2023"
+author: Mikaela De Smedt
+date: 2023-12-16
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
+# Introduction
```

Comment:
Use level two headings `##` as the first level. Level one corresponds to the project title that is generated from the header of the file.